backend/app/services/db_service.py

```
@@ -0,0 +1,326 @@
+                SELECT table_name 
+                FROM information_schema.tables 
+                WHERE table_schema = 'public' 
+                AND table_type = 'BASE TABLE'
+            """)
+            
+            if tables:
+                table_names = [table['table_name'] for table in tables]
+                
+                # CASCADEで全削除
+                for table_name in table_names:
+                    await self.db.execute(f'DROP TABLE IF EXISTS "{table_name}" CASCADE')
```

Flake8 (7.2.0)

[error] line too long (89 > 79 characters)

(E501)